content/docs/2.10/concepts/_index.md

```diff
@@ -12,10 +12,11 @@ weight = 1
 
 ## How KEDA works
```

Comment:
A nit on the new overview - All KEDA components are grouped, except for the  new one. IMO we should move it.

![image](https://user-images.githubusercontent.com/4345663/210948182-28896a6d-8ef9-4f55-961f-66d1d210b801.png)

Reply:
I placed out because conceptually it's different from other pieces, but I can update the graphic and place it at the same level.

Reply:
done!